script/hassfest/services.py

```diff
@@ -33,6 +33,10 @@ def exists(value: Any) -> Any:
         vol.Optional("required"): bool,
         vol.Optional("advanced"): bool,
         vol.Optional(CONF_SELECTOR): selector.validate_selector,
+        vol.Optional("filter"): {
+            vol.Optional("supported_features"): [int],
+            vol.Optional("supported_color_modes"): [str],
```

Comment:
This feels weird to have this be an option for all services of all integrations, as it's only necessary for the `light` services.

Can we have some attribute filter, where we check if an attribute is part of a specific value?

Reply:
attribute would be `color_modes`

Reply:
Right, we use `options` for the select selector. I'm not sure why the attribute would be `color_modes` though; the name of the attribute is `supported_color_modes`?